index.bs

```diff
@@ -4401,6 +4401,11 @@ for {{DOMException}}, a description, and legacy code values.
 
 Note: If an error name is not listed here, please file a bug as indicated at the top of this specification and it will be addressed shortly. Thanks!
 
+Note: Do not confuse {{SyntaxError!!exception}} defined here with ECMAScript's
+{{ECMAScript/SyntaxError}}. The ECMAScript one is deliberately omitted from
```

Comment:
I'd write something like this again here:

> The ECMAScript {{ECMAScript/SyntaxError}} is deliberately omitted from [=simple exceptions=] as it is reserved for the ECMAScript parser.

And then add something about using "{{SyntaxError!!exception}}" {{DOMException}} for other kids of parsing erros (e.g. CSS parser) after that.